charts/stable/sickgear/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: "0.25.4"
+appVersion: "0.25.60"
```

Comment:
It looks like maybe this should be "latest" since nothing else seems to bump this later on?

Reply:
Moved this to latest as that's what other examples (e.g. valheim) use.